README.md

```diff
@@ -117,22 +117,26 @@ The command should return the absolute path to the binary if it's found in the s
 
 How to configure Neovim with lua modules: https://neovim.io/doc/user/lua-guide.html#lua-guide-config
 
-Use lspconfig to create a custom contextive language server configuration and initialize the language server by calling the setup function. The following lua snippet needs to be included in the `init.lua` file either directly or from another lua module like `lspconfigs.lua`.
+Contextive Language Server is also provided with [Mason](https://github.com/williamboman/mason.nvim).
+
+Use lspconfig to setup and initialize the Contextive Language Server configuration. The following lua snippet needs to be included in the `init.lua` file either directly or from another lua module like `lspconfigs.lua`.
```

Comment:
Worth linking 'lspconfig' to either https://github.com/neovim/nvim-lspconfig or https://luarocks.org/modules/neovim/nvim-lspconfig here?

I'm not sure in the neovim community if lspconfig is something everyone will already have or if we should guide them on how to get it first?

Reply:
That's a good point! I linked the neovim/lsp-config installation to this document! It is required to be installed as a _plugin_.